Src/AutoFixture.NUnit3.UnitTest/Scenario.cs

```diff
@@ -295,5 +295,52 @@ public void FreezeSecondParameterOnlyFreezesSubsequentParameters(Guid g1, [Froze
         {
             Assert.AreNotEqual(p1, p2.Field);
         }
+
+        [Theory]
+        [InlineAutoData(1, 2, 3)]
+        public void InlineAutoDataTakesParameterValues(int p1, int p2, int p3)
+        {
+            Assert.That(1, Is.EqualTo(p1));
+            Assert.That(2, Is.EqualTo(p2));
+            Assert.That(3, Is.EqualTo(p3));
+        }
+
+        [Theory]
+        [InlineAutoData]
+        public void InlineAutoDataProvidesParameterValuesWhenNoneGiven(int p1, int p2, int p3)
+        {
+            Assert.That(0, Is.Not.EqualTo(p1));
+            Assert.That(0, Is.Not.EqualTo(p2));
+            Assert.That(0, Is.Not.EqualTo(p3));
+        }
+
+        [Theory]
+        [InlineAutoData(1, 2)]
+        public void InlineAutoDataProvidesParameterValuesWhenMissing(int p1, int p2, int p3)
+        {
+            Assert.That(1, Is.EqualTo(p1));
+            Assert.That(2, Is.EqualTo(p2));
+            Assert.That(0, Is.Not.EqualTo(p3));
+        }
+
+        [Theory]
+        [InlineAutoData(1, 2)]
+        [InlineAutoData(4)]
+        [InlineAutoData]
+        public void InlineAutoDataCanBeUsedInMultiple(int p1, int p2, int p3)
+        {
+            Assert.That(0, Is.Not.EqualTo(p1));
+            Assert.That(0, Is.Not.EqualTo(p2));
+            Assert.That(0, Is.Not.EqualTo(p3));
+        }
```

Comment:
[Same](https://github.com/AutoFixture/AutoFixture/pull/589/files#r57543945) here as well.

In addition to that, how do we know that there are multiple test-runs? Shouldn't NUnit3 be responsible for this outcome?